### PR TITLE
CVE 2022 3171

### PR DESCRIPTION
**Pull request by kaushalmahi12**
**Is your feature request related to a problem? Please provide an existing Issue # , or describe.**
This change is to address the [issue](https://github.com/opensearch-project/performance-analyzer/issues/294)

**Describe the solution you are proposing**
Upgrading the **protobuf-java** to suggested version.

**Describe alternatives you've considered**

**Additional context**

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/performance-analyzer/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -281,7 +281,7 @@ dependencies {
     implementation(group: 'com.google.errorprone', name: 'error_prone_annotations', version: '2.9.0') {
         force = 'true'
     }
-    implementation(group: 'com.google.protobuf', name:'protobuf-java', version: '3.19.2') {
+    implementation(group: 'com.google.protobuf', name:'protobuf-java', version: '4.0.0-rc-1') {
```
Comment (kkhatua) [inline build.gradle:284]: Should we be using an RC as a dependency??

**Comment by kkhatua** [comment]
Do we need to bump version for RCA package as well?

File: build.gradle (hunk 1 above)
Comment (kaushalmahi12) [reply build.gradle:284]: It is the recommended in the suggested fix.

**Comment by kaushalmahi12** [comment]
> Do we need to bump version for RCA package as well?

I think yes, because both packages are dependent on each other.

**Review by kkhatua — APPROVED** [review]

**Review by dzane17 — APPROVED** [review]

**Comment by kaushalmahi12** [comment]
Closing this PR as [CVE 42003] already has the changes from this PR.

**Pull request closed without merging**